qfdmo/management/commands/update_external_ids.py

```diff
@@ -53,6 +53,22 @@ def handle(self, *args, **options):
                 )
                 continue
 
+            # test acteur with this new external id already exists
+            acteur_from_db = Acteur.objects.filter(
+                identifiant_externe=new_id, source__code=source_code
+            ).first()
+            id_index = 0
+            while acteur_from_db:
+                id_index += 1
+                new_id_indexed = f"{new_id}_{id_index}"
+                acteur_from_db = Acteur.objects.filter(
+                    identifiant_externe=new_id_indexed, source__code=source_code
+                ).first()
+            statut = ActeurStatus.ACTIF
+            if id_index:
+                new_id = f"{new_id}_{id_index}"
+                statut = ActeurStatus.INACTIF
```

Comment:
```suggestion
            try new_id_indexed;
                new_id = new_id_indexed
                statut = ActeurStatus.INACTIF
            except NameError:
                pass
```

pour être plus [pythonesque](https://docs.python.org/3/glossary.html#term-EAFP)